src/components/CategoryCard.jsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+
+export function CategoryCard({ category }) {
```

Comment:
category can be destructed more here
`{ category: { imageSrc, text, subText}}`

Reply:
Done